rollup.config.js

```diff
@@ -40,6 +40,18 @@ const config = merge(
   createSpaConfig({
     outputDir: DIST_PATH,
     legacyBuild: true,
+    polyfillsLoader: {
```

Comment:
Probably I'm wrong here, @abdonrd but it's this polyfill needed now?

Reply:
You can check here the Lit docs:
https://lit.dev/docs/releases/upgrade/#load-polyfill-support-when-using-web-components-polyfills